scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 574,
-  MAX_SIZE: 877,
+  MAX_SIZE: 891,
```

Comment:
Thats quite an increase in bundle size :(

Reply:
Yeah it came to 886 when I integrated the banner, but now it's 884 for me locally